### DISCUSSION TITLE
Is it possible to publish a web server based version?

### DISCUSSION THREAD

**Discussion by xinbe** [alagrede/znote-app#61] [Ideas]
Is it possible to publish a web server based version?

**Comment by alagrede** [comment]
Hi @xinbe,
Thanks for your feedback 😊 
Could you tell me more about your need or your use-case (mobile/desktop). Do you want edit your notes without install the app or maybe publicly expose your notes?

One of the main capabilities of Znote is to execute code blocks of JS and use local installed NPM packages and Node API to read local files. Currently I don't plan to move this feature into a web version due to all the infrastructure constraints behind it.
But it is possible to put in my roadmap a **lighter web version for the next major version**.

Have you seen we have a Znote IOS/Android mobile app?
Hoping to read you soon.

Cordially
Anthony
